src/styles/application/_errorScreen.scss

```diff
@@ -0,0 +1,49 @@
+.integr8ly-errorscreen {
+  display: flex;
+  flex-direction: column;
+
+  &-backdrop {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background-size: cover;
+    background-color: $pf-color-blue-600;
+    background-image: url("../../img/provisioning-bg.png");
+    z-index: -5;
```

Comment:
A negative z-index should be avoided. From what I can see, if this error page is reused inside of the content area, it would collide with the fixed elements (thinking ahead to the dynamic/framed approach).

If the masthead is causing an issue, you can change the top positioning of the background to `40px`. (`top: 40px;`). Other adjustments can be made at a later date to deal with the problematic masthead positioning.